tests/test_characters_counter.py

```diff
@@ -5,21 +5,27 @@
 
 class TestCharactersCounter:
     @pytest.mark.parametrize(
-        'input_text, expected_result',
-        [("abbbcccdf", 3), ("aabbccdd", 0),
-         ("!@#123abcdee", 10), ("", 0),
-         ("aAbBcC", 6),]
+        "input_text, expected_result",
+        [
+            ("abbbcccdf", 3),
+            ("aabbccdd", 0),
+            ("!@#123abcdee", 10),
+            ("", 0),
+            ("aAbBcC", 6),
+        ],
     )
     def test_count_characters_occurrence(
-            self, input_text, expected_result) -> None:
+        self, input_text: str, expected_result: str
+    ) -> None:
         assert count_characters_occurrence(input_text) == expected_result
 
-    @pytest.mark.parametrize('input_text', [123, 30.6, [], {}, set()])
+    @pytest.mark.parametrize("input_text", [123, 30.6, [], {}, set()])
     def test_count_characters_occurrence_raises_TypeError_when_unhashable_input_is_provided(
-            self, input_text) -> None:
+        self, input_text: str
+    ) -> None:
         with pytest.raises(TypeError):
             count_characters_occurrence(input_text)
 
 
-if __name__ == '__main__':
+if __name__ == "__main__":
     pytest.main()
```

Comment:
Please add github badge into readme

https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/monitoring-workflows/adding-a-workflow-status-badge

Reply:
I added badge, thank you!